afqbrowser/site/client/js/sortable-table.js

```diff
@@ -300,25 +300,25 @@ afqb.table.refreshTable = function () {
             // numGroups may be smaller if there are not enough unique values
             var usrGroups = afqb.table.settings.sort.count;
             var numGroups = Math.min(usrGroups, uniques.length);
-			var groupScale;
-            
+						// var groupScale;
+
             // Create groupScale to map between the unique
             // values and the discrete group indices.
             // TODO: Use the datatype json instead of
             // just testing the first element of uniques
             if (typeof uniques[0] === 'number') {
                 if (afqb.table.settings.splitMethod === "Equal Size" || numGroups === 1) {
```

Comment:
Responding to your question on #204. 
Here we use different d3 scales depending on the `splitMethod` and data type.

- If the column data is a number, we use either [`d3.scale.quantile()`](https://github.com/d3/d3-3.x-api-reference/blob/master/Quantitative-Scales.md#quantile) or [`d3.scale.quantize()`](https://github.com/d3/d3-3.x-api-reference/blob/master/Quantitative-Scales.md#quantize). In either of these cases, you can use the `invertExtent()` method to get the bin threshold, feeding it the group number as input.
- If the column data is not a number, then we use [`d3.scale.ordinal()`](https://github.com/d3/d3-3.x-api-reference/blob/master/API-Reference.md#ordinal). In this case, I'm not sure what the tooltip should show since the bins are pretty arbitrary (e.g. alphabetical binning on the subjectID string). Might be best not to write the bin thresholds in this case and just let the user infer from looking at the colors and the table.